src/main/java/org/codeforamerica/shiba/application/parsers/ApplicationDataParserV2.java

```diff
@@ -0,0 +1,66 @@
+package org.codeforamerica.shiba.application.parsers;
+
+import lombok.Getter;
+import org.codeforamerica.shiba.pages.data.PagesData;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ApplicationDataParserV2 {
+
+    /**
+     * Retrievable fields
+     */
+    public enum Field {
```

Comment:
Should this be defined outside of this class?

Reply:
Given how general the name is I think it's ok for it to be inline.